lpcompiler.pas

```diff
@@ -1310,7 +1324,10 @@ function TLapeCompiler.ParseMethodHeader(out Name: lpString; addToScope: Boolean
         end;
       until (Tokenizer.Tok = tk_sym_ParenthesisClose);
 
-    if isFunction then
+    if Result.isOperator and (Result.Params.Count <> 2) then
```

Comment:
Can we maybe check here or in `ParseMethod` whether or not it is allowed to define this function (for example, if it's not overriding a built-in operator)?